### ISSUE TITLE
Kontraste und Anpassung von Größenverhältnissen optimieren 

### ISSUE THREAD

**Issue by lisaschreyer** [wirDesign-communication-AG/wirhub-app#1228]
Aufwände: 4h Entwicklung | 4h Konzeption und UI/UX-Design

# Anforderung: 
Bedienelemente und Text innerhalb der Seitenbausteine (z.B. Farben) (nicht Content) müssen Kontraste gem. Barrierefreiheitsrichtlinien einhalten

### Lösung:
- Designoptimierungen innerhalb der SmartBlocks, damit Kontraste eingehalten werden
- Größenverhältnisse einiger Bedienelemente optimieren

### Betrifft: 
- Farben
- Slideshow
- Hinweis

# Design ToDos
- [x] Warnfarbe neu definieren wirHub Theme → Richtlinien für Kundenthemes festlegen
- [x] SB Slideshow: Neue Position Pfeile (Auch für SB Neuigkeiten mitdenken)
- [x] Kontrast in Farbabstufungen (SB Farben)

# Development ToDos

### Update default Support Farben:

- [x] $color-warning-dark: #F75C03
- [x] $color-warning-light: #FFE6D5

### Smart-Block Slideshow:
**Navigationspfeile:**
- [x] Position: nach unten, auf die gleiche Höhe wie die navigation dots
- [x] Größe: 32px hoch
- [x] Farbe: neutrals-black

**Navigation Dots:**
Farben:
- [x] inactive: neutrals-40
- [x] active: neutrals-black

Größe:
- [x] inactive: 15px
- [x] active 20px

**Design:** 
https://www.figma.com/design/kpIVnqrNdJhZL1eAoEf2hm/Smart-Blocks?node-id=3445-12052&t=RLuadDVlodV4I26s-1

### Smart-Block Farben
- [x] die Prozentwerte bei den Abstufungen unter das Farbfeld setzen
**Design:** 
https://www.figma.com/design/kpIVnqrNdJhZL1eAoEf2hm/Smart-Blocks?node-id=3981-6264&t=RLuadDVlodV4I26s-1

**Comment by lisaschreyer** [comment]
Ergänzung LS1: 
Hier haben wir besprochen, dass wir beim Hinweis aus gelb als Basis ein orange machen und statt primary 60 bei info primary 50 oder 40 nehmen. 
Kommt aber auch immer auf das Kundendesign an. 
Bei Slideshow schieben wir nur die Pfeile nach unten, aus dem Bild raus.
Bei Farben haben wir gesagt, dass das ein extremer Edge Case ist, weil das praktisch grade nur K** so benutzt eigentlich.
